content/en/docs/concepts/deployment-models/index.md

```diff
@@ -0,0 +1,421 @@
+workload instances. A single deployment of the control plane consists of several
+redundant processes that share a common view of the mesh and provide workload
+instances with identical configuration.
+
+Depending on your overall system architecture and cloud provider, the following
+Istio control plane models are possible:
+
+- [single control plane](#single-control-plane)
+- [multiple control planes](#multiple-control-planes)
+- [managed control planes](#managed-control-plane)
+
+### Single control plane
+
+In the simplest case, you can run your mesh using a single control plane
+deployment. The single control plane model is typically used for [single
+cluster](#single-cluster) meshes.
+
+If you run your mesh on a single cluster, a single control plane deployment
+suffices. Each workload instance in the cluster connects to a control plane
+instance to get its configuration.
+
+{{< image width="50%"
+    link="./single-cluster.svg"
+    alt="A service mesh with a single control plane"
+    title="Single control plane"
+    caption="A service mesh with a single control plane"
+    >}}
+
+### Multiple control planes
+
+You can deploy multiple control planes across the mesh; such a configuration is
+known as a **multi-control plane**.
+
+Multiple control planes afford improved performance and availability for meshes
+that span [multiple clusters](#multiple-clusters) across multiple zones and
+regions.
+
+Istio supports control plane deployments at various levels of availability:
+
+- Control plane per region: *lowest availability*
+- Control plane per zone
+- Control plane per cluster *highest availability*
+
+The configuration of each control plane could be identical or different
+depending on the [network](#network-models) and [cluster](#cluster-models)
+models of the system.
+
+{{< image width="50%"
+    link="./multi-control.svg"
+    alt="A service mesh with multiple control planes"
+    title="Multiple control planes"
+    caption="A service mesh with multiple control planes"
+    >}}
+
+### Managed control plane
+
+Many cloud providers offer a control plane, which they manage on behalf of their
+customer.
+
+Typically, managed control planes guarantee some level of performance and
+availability, which you can assume when building your system. Managed control
+planes affords the following benefits:
+
+- Greatly reduce the complexity of user deployments
+- Greatly reduce system administration costs
+- Effectively eliminate the usage costs associated with the control plane:
+    - CPU
+    - Memory
+    - Network
+
+{{< image width="50%"
+    link="./managed-control.svg"
+    alt="A service mesh with a managed control plane"
+    title="Managed control plane"
+    caption="A service mesh with a managed control plane"
+    >}}
+
+## Identity and trust models
+
+When a workload instance is created within a service mesh, Istio assigns the
+workload an identity. An identity is a unique name backed by a key material, for
```

Comment:
@smawson if the name, itself, is the identity then it's not cryptographically signed, is it? Rather, it's backed by something that is signed (e.g. cert). With SPIFFE, for example, the identity would be the SPIFFE ID (which is just the sa name). The SVID, however, (which contains the ID) is the signed and verifiable "thing". So which is the identity in this case? The SVID or just the ID? 

I think we need to get this definition right here ... otherwise it will be a source of confusion.

Reply:
Righto, the identity is a trust domain, namespace, and service account. The SVID is a representation of that identity, and the cert is a signed attestation of that identity. But fundamentally the identity is just the tuple of those pieces, with lots of representations.